creator-node/src/services/stateMachineManager/stateReconciliation/SecondarySyncHealthTracker.ts

```diff
@@ -147,15 +168,19 @@ const Getters = {
    * @returns {Object} { '0x...': { 'https://secondary1...': { 'successCount' : _, 'failureCount': _, 'successRate': _ }, ... } ... }
    */
   async computeUsersSecondarySyncSuccessRatesForToday(
-    walletsToSecondariesMapping
-  ) {
+    walletsToSecondariesMapping: WalletsToSecondariesMapping
+  ): Promise<UserSecondarySyncMetricsMap> {
     // Initialize sync success and failure counts for every secondary to 0
-    const secondarySyncMetricsMap = {}
+    const secondarySyncMetricsMap: UserSecondarySyncMetricsMap = {}
     const wallets = Object.keys(walletsToSecondariesMapping)
     for (const wallet of wallets) {
       const secondarySyncMetrics = secondarySyncMetricsMap[wallet] || {}
       for (const secondary of walletsToSecondariesMapping[wallet]) {
-        secondarySyncMetrics[secondary] = { successCount: 0, failureCount: 0 }
+        secondarySyncMetrics[secondary] = {
+          successCount: 0,
+          failureCount: 0,
+          successRate: 1
+        }
```

Comment:
this plus using the import from state machine instead of snapback is what might possibly fix https://linear.app/audius/issue/CON-280/fix-cannot-convert-undefined-or-null-to-object-bug